src/test/java/mat/server/CQLConstantServiceImplTest.java

```diff
@@ -0,0 +1,50 @@
+package mat.server;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import mat.client.shared.CQLConstantContainer;
+import mat.config.DaoTestConfig;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ContextConfiguration(classes = DaoTestConfig.class)
```

Comment:
I believe that ContextConfiguration and MocitoExtension annotations are not compatible.
Most likely 'ContextConfiguration' is not actually used. You just inject all mocks and unresolved fhirMatServicesUrl